### PR TITLE
[feature](index) Replace BITMAP INDEX with INVERTED INDEX

### PR DESCRIPTION
**Pull request by xiaokang**
## Proposed changes

It's time to replace BITMAP INDEX with INVERTED INDEX, since INVERTED INDEX is production ready in 2.0 version.

For compatibility, 
1. already created BITMAP INDEX can still be used BUT users can not create new BITMAP INDEX any more. 
2. If the user use the `USING BITMAP INDEX` to create index, a INVERTED INDEX is created actually, to avoid error and get a similar query acceleration.

For the further version, `BITMAP INDEX` will be completely removed.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xiaokang** [comment]
run buildall

**Review by airborne12 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java [hunk 1]
```diff
@@ -2527,6 +2527,10 @@ public IndexDefinition visitIndexDef(IndexDefContext ctx) {
         Map<String, String> properties = visitPropertyItemList(ctx.properties);
         String indexType = ctx.indexType != null ? ctx.indexType.getText().toUpperCase() : null;
         String comment = ctx.comment != null ? ctx.comment.getText() : "";
+        // change BITMAP index to INVERTED index
+        if (indexType.equals("BITMAP")) {
```
[the remark this answers is not in the document]
Comment (xiaokang) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java]: done

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Pull request merged**